src/ripple/beast/unity/beast_hash_unity.cpp

```diff
@@ -18,9 +18,7 @@
 //==============================================================================
 
 #include <ripple/beast/hash/xxhasher.h>
-#include <ripple/beast/hash/impl/spookyv2.cpp>
 
 #if ! BEAST_NO_XXHASH
```

Comment:
I think `BEAST_NO_XXHASH` might be unsupported now?  I tried building with it set and got several errors, so I think it's not an option any more.

Reply:
Good catch!